src/components/Router/StepsRouter.js

```diff
@@ -18,16 +21,26 @@ class StepsRouter extends Component {
 
   currentComponent = () => this.props.componentsList[this.props.step]
 
+  useFullScreen = (value) => this.setState({fullScreen: value})
+
+  hasLiveCaptureOption = (options) => options && options.liveCapture
```

Comment:
is this needed?

Reply:
This is test code that I have added to facilitate acceptance. Will get rid of it.